server/api/after_auth_handler.py

```diff
@@ -29,7 +30,9 @@ def authorize():
         Then db_user would be object containing
         property inserted_id
         """
-        db_user = User(g_user).save("users")
+        myUser = User(g_user)
```

Comment:
just a note. Python naming conventions are CapWords for classes, but snake_case for pretty much everything else. And pretty much never camelCase
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names